### DISCUSSION TITLE
Pop out the coverage table (show the coverage result on an own page)

### DISCUSSION THREAD

**Discussion by ghost** [browserslist/browserslist#793] [Ideas]
At the moment we are getting this page …

---
<img width="1219" alt="screenshot" src="https://github.com/browserslist/browserslist/assets/2291224/8a92613e-5d0c-4c5e-bd6d-ad04dad93e4d">

---

… but it would be nice, if we could get the right column (optionally) as an own page, so that we can send it to our client, without the clutter on the left column.

This would be super nice! What do you think?

Upvote this feature!
↓

**Comment by ai** [comment]
We can add URL like `https://browsersl.ist/?results#q=defaults` and hide left column by JS on `?results` in this case.

Maybe we will need buttons `Hide docs`/`Show docs` to show this feature to users.

**Reply by ghost** [reply]
Or as soon as the result is shown up, the docs are hidden und a "show docs" button appears … makes sense?

**Comment by ai** [comment]
Done https://github.com/browserslist/browsersl.ist/commit/1e86df8dc63a624f0687d37053cc2169af7a43d3

Example: https://browsersl.ist/?results=#q=defaults

**Reply by ghost** [reply]
Nice! Thank you a lot for this and everything else you are doing for the community!
